### ISSUE TITLE
Control port shows bad on Rev2 harness

### ISSUE THREAD

**Issue by Obiwantje** [svenpetersen1965/C64-Diagnostic-Rev.-586220-Harness#7]
Sven, got several of your test Harness builds (in all versions) - but this latest non-SMD version keeps on giving me  bad control port on all C64 boards (I have many) - I have replaced the 316Ohm resistor with a pot so I could dial in exactly 316, but to no avail - I have no clue as to why I can't seem to get this version of the harness to not show this (consistent) issue on all boards.....any ideas of what i could try out?

**Comment by svenpetersen1965** [comment]
That could have several reasons.
1. the ribbon cable(s) might be bad. I have experienced problems with one (rainbow) ribbon cable. It was not in the 1.27mm pitch. Adrian Black had this issue, too.
2. The 110k resistors are not 110k. They are used to simulate the pots and if the result is not within a certain  range, the control port is reported bad. 
I don’t believe, that it is an issue with the 316 ohm resistor. The value is not critical. I have picked it, because it was closest to the 320 ohms from the original harness schematics. 330 ohms have proved to work well.
Greetings,
Sven
